macros/base/park.cfg

```diff
@@ -51,3 +51,19 @@ gcode:
         G1 Z{z_safe} F{Sz}
         G0 X{Px} Y{Py} F{St}
     {% endif %}
+
+
+[gcode_macro PARK_FRONT]
+description: Park the toolhead on the front of the printer for maintenance
+gcode:
+    {% set St = printer["gcode_macro _USER_VARIABLES"].travel_speed * 60 %}
+    {% set Sz = printer["gcode_macro _USER_VARIABLES"].z_drop_speed * 60 %}
+
+    {% if "xyz" not in printer.toolhead.homed_axes %}
+        G28   ; home if not already homed
```

Comment:
I would rather do it same as original PARK behaviour and not have two different PARKs behaving differently on homed axis.

Reply:
OK for adding a house to `PARK`,
`PARK` is not really well coded, because if you add the `E` parameter when you call `PARK`, it is replaced by the default value from the restract hardware or firmware configuration, which doesn't make sense. If you force a parameter in the call, you have to apply it.
This should be part of another PR that would make `PARK` cleaner

Reply:
well, this changes it. 
Park was: Not homed -> error
Park_front was: Not homed -> homing

instead of 
park_front: Not homed -> error

it is not 
park: not homed -> homing.

I am not sure which way is the better one.
Guess something for @Frix-x to decide.

Reply:
It's ok for me as is. We will work on improving it in another PR if needed